tests/python/test_multidevice.py

```diff
@@ -34,34 +34,27 @@ def test_pointwise(multidevice_test):
 
     torch.cuda.set_device(multidevice_test.local_rank)
 
-    # Inputs
-    inputs = [
-        torch.randn(num_devices, 4, device="cuda")[rank : rank + 1, ...],
-    ]
+    # Just so each rank receives the same unsharded input.
+    torch.manual_seed(0)
+    unsharded_input = torch.randn(num_devices, 4, device="cuda")
+    sharded_input = unsharded_input[rank : rank + 1]
 
-    # dynamic shape isn't supported;
-    # scalar inputs isn't supported;
     class MultiDeviceModel(FusionDefinition):
         def definition(self):
             self.t0 = self.define_tensor(
-                (num_devices, 4), (False, False), dtype=DataType.Float
+                (-1, -1), contiguity=(False, False), dtype=DataType.Float
```

Comment:
Does this work with `-np 1`?

Not that we have to have that in the test, but I thought the CI script has that... (I could be misreading that)

Reply:
NVM, this seems to work fine. :clap: